decidim-core/spec/system/homepage_spec.rb

```diff
@@ -98,7 +98,7 @@
 
           it "uses the custom values for the CTA button text" do
             within ".hero" do
-              expect(page).to have_selector("a.hero-cta", text: "SIGN UP")
+              expect(page).to have_selector("#hero-cta", text: "SIGN UP")
```

Comment:
are/will we be able to have more than one hero? if so, would this work?

Reply:
I seriously doubt it. Two heroes in the homepage is... kinda overkilling, isn't? Currently it's not possible, is the idea being able to deal with more?

Reply:
With the just added feature its possible to have N blocks of each type. You can test it in our staging. 

<img width="1256" alt="CleanShot 2023-02-01 at 09 53 56@2x" src="https://user-images.githubusercontent.com/24740/215996106-84dd162f-bdc8-4f75-be2b-50607dfc3348.png">

<img width="1260" alt="CleanShot 2023-02-01 at 09 54 23@2x" src="https://user-images.githubusercontent.com/24740/215996140-263cdc4a-889d-4bf2-9f62-843ad6213fe2.png">

Reply:
After a talk with @entantoencuanto we agreed to simplify the test. Besides, when the content-block integration goes forward, he will have to adapt some code, including the tests, therefore, no issue in the meanwhile